applications/matter_bridge/doc/matter_bridge_description.rst

```diff
@@ -733,21 +733,12 @@ See :ref:`app_build_file_suffixes` and :ref:`cmake_options` for more information
 Configure the functionality of the Matter-Bridge device
 -------------------------------------------------------
 
-To enable the Matter smart plug functionality, run the following command:
+To enable the Matter smart plugin functionality, for example with :ref:`zephyr:nrf54h20dk_nrf54h20`, run the following command:
 
-.. tabs::
-
-   .. group-tab:: nRF54 DKs
-
-      .. code-block:: console
-
-         west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
-
-   .. group-tab:: nRF70 DKs
-
-      .. code-block:: console
+.. parsed-literal::
+   :class: highlight
 
-         west build -b nrf7002dk/nrf5340/cpuapp -p -- -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
+   west build -b nrf54h20dk/nrf54h20/cpuapp -p -- -DSB_CONFIG_WIFI_NRF700X=y -DCONFIG_CHIP_WIFI=y -Dmatter_bridge_SHIELD=nrf700x_nrf54h20dk -DCONFIG_BRIDGED_DEVICE_BT=y -Dmatter_bridge_SNIPPET=onoff_plug
```

Comment:
And with regards to the above comment:
```suggestion
   west build -b *board_target* -p -- -Dmatter_bridge_SNIPPET=onoff_plug
```

Reply:
@kg-nordicsemi why haven't you applied this comment and resolved conversation? I've just noticed that the documentation was not fixed while reviewing the guide before the release. I suggest to resolve conversation AFTER applying the fix to not forget to do that.

Reply:
Sorry, it wasn't your fault :) Someone probably didn't rebase after your commit was merged and then force pushed PR, so your change was removed: https://github.com/nrfconnect/sdk-nrf/commit/b364fd802a36bd39cde9202d65735da8677bae97#diff-f250f5581599929b90c895f576af0b1ccf1b6dc333c1d6a859ac992cae861eb9R740

I will fix this.